src/javaRosa/util.js

```diff
@@ -281,6 +281,11 @@ define([
                     return $this[0].outerHTML;
                 });
                 text = xml.normalize(text.html());
+                if (escape) {
+                    text = text.replace(/(<)|>/g, function (match, lt) {
+                        return lt ? "&lt;" : "&gt;";
+                    });
```

Comment:
Replacing multiple occurrences of a string in JS requires a regexp (with the `g` flag).